sv_pipeline/genome/load_data.py

```diff
@@ -0,0 +1,211 @@
+import argparse
+import hail as hl
+import logging
+import os
+import time
+
+from hail_scripts.v02.utils.elasticsearch_client import ElasticsearchClient
+
+from sv_pipeline.utils.common import get_sample_subset, get_sample_remap, get_es_index_name, CHROM_TO_XPOS_OFFSET
+from sv_pipeline.genome.utils.mapping_gene_ids import load_gencode
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+EXP_CHROM_TO_XPOS_OFFSET = hl.literal(CHROM_TO_XPOS_OFFSET)
+
+WGS_SAMPLE_TYPE = 'WGS'
+
+INTERVAL_TYPE = 'array<struct{type: str, chrom: str, start: int32, end: int32}>'
+
+BASIC_FIELDS = {
+    'contig': lambda rows: rows.locus.contig.replace('^chr', ''),
+    'sc': lambda rows: rows.info.AC[0],
+    'sf': lambda rows: rows.info.AF[0],
+    'sn': lambda rows: rows.info.AN,
+    'start': lambda rows: rows.locus.position,
+    'end': lambda rows: hl.if_else(hl.is_defined(rows.info.END2), rows.info.END2, rows.info.END),
+    'sv_callset_Het': lambda rows: rows.info.N_HET,
+    'sv_callset_Hom': lambda rows: rows.info.N_HOMALT,
+    'gnomad_svs_ID': lambda rows: rows.info.gnomAD_V2_SVID,
+    'gnomad_svs_AF': lambda rows: rows.info.gnomAD_V2_AF,
+    'pos': lambda rows: rows.locus.position,
+    'filters': lambda rows: hl.array(rows.filters.filter(lambda x: x != 'PASS')),
+    'xpos': lambda rows: get_xpos(rows.locus.contig, rows.locus.position),
```

Comment:
You are right. The `xpos` needs to be annotated before annotating `xstart` so that its value can be reused by `xstart`. It could be annotated separately.  But I would like to put all the output fields in either `BASIC_FIELDS` or `COMPUTED_FIELDS` which are neat. Maybe the name of `COMPUTED_FIELDS` is confusing. It could be changed to `ADDITIONAL_FIELDS`.